pkg/kube/apisix/apis/config/v2beta2/types.go

```diff
@@ -195,3 +195,40 @@ type ApisixRouteList struct {
 	metav1.ListMeta `json:"metadata" yaml:"metadata"`
 	Items           []ApisixRoute `json:"items,omitempty" yaml:"items,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+
+// ApisixPluginConfig is the Schema for the ApisixPluginConfig resource.
+// An ApisixPluginConfig is used to support a group of plugin configs
+type ApisixPluginConfig struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata" yaml:"metadata"`
+
+	// Spec defines the desired state of ApisixPluginConfigSpec.
+	Spec   ApisixPluginConfigSpec `json:"spec" yaml:"spec"`
+	Status v2alpha1.ApisixStatus  `json:"status,omitempty" yaml:"status,omitempty"`
```

Comment:
I think the reference from `v2beta2` to `v2alpha1` is not so elegant. Of course it doesn't affect this PR's review.

cc @tao12345666333 .

Reply:
I ignored it before, i will update after next commit